### PR TITLE
feat: add bucket env var for storing artifact in a single bucket

### PR DESCRIPTION
**Pull request by nicufk**
## Pull request description 

Related to https://github.com/kubeshop/testkube/issues/2940

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: charts/testkube-api/templates/deployment.yaml [hunk 1]
```diff
@@ -90,6 +90,12 @@ spec:
               {{- else }}
               value:  "testkube-minio-service-{{ .Release.Namespace }}:{{ .Values.storage.endpoint_port }}"
               {{- end }}
+            - name: "STORAGE_BUCKET"
+            {{- if .Values.storage.bucket }}
+              value:  "{{ .Values.storage.bucket }}"
+            {{- else }}
+              value:  "testkube-{{ uuidv4 }}"
```
Comment (vsukhin) [inline charts/testkube-api/templates/deployment.yaml:97]: will it be the same for each update?

**Review by vsukhin — APPROVED** [review]
one question

File: charts/testkube-api/templates/deployment.yaml (hunk 1 above)
Comment (nicufk) [reply charts/testkube-api/templates/deployment.yaml]: Removed the uuid, user will have to take care for AWS S3 to provide a unique name

Comment (vsukhin) [reply charts/testkube-api/templates/deployment.yaml]: Might be to use "testkube-{{ .Release.Namespace }"  by default?

**Review by vLia — APPROVED** [review]

**Pull request merged**